pkg/ocm/invite/invite.go

```diff
@@ -18,6 +18,22 @@
 
 package invite
 
-// Manager is the interface that is used to perform operatuons to invites.
+import (
+	"context"
+
+	userpb "github.com/cs3org/go-cs3apis/cs3/identity/user/v1beta1"
+	invitepb "github.com/cs3org/go-cs3apis/cs3/invite/v1beta1"
+	ocm "github.com/cs3org/go-cs3apis/cs3/sharing/ocm/v1beta1"
+)
+
+// Manager is the interface that is used to perform operations to invites.
 type Manager interface {
+	// GenerateToken creates a new token for the user with a specified validity.
+	GenerateToken(ctx context.Context, user *userpb.UserId) (*invitepb.InviteToken, error)
```

Comment:
You don't need the user *userpb.UserId argument here, the implementation will be able to obtain it from the context, this action is protected with the authentication middleware.

Reply:
Yep. Got it :+1: